uservice_ccutter/server.py

```diff
@@ -215,7 +215,7 @@ def create_github_repository(auth, userdict):
         ghub = github3.login(auth["username"], token=auth["password"])
         try:
             ghub.me()
-        except github3.exceptions.AuthenticationFailed:
+        except (github3.exceptions.AuthenticationFailed, AttributeError):
```

Comment:
I'm not using it as an "as"; I'm catching either of the two types of exceptions.